google/cloud/aiplatform/docker_utils/run.py

```diff
@@ -110,6 +113,18 @@ def run_prediction_container(
         host_port (int):
             Optional. The port on the host that the port, AIP_HTTP_PORT, inside the container
             will be exposed as. If it's unset, a random host port will be assigned.
+        gpu_count (int):
+            Optional. Number or devices to request. Set to -1 to request all available devices.
+            To use GPU, set either `gpu_count` or `gpu_device_ids`.
+        gpu_device_ids (List[str]):
+            Optional. List of strings for device IDs.
+            To use GPU, set either `gpu_count` or `gpu_device_ids`.
+        gpu_capabilities (List[List[str]]):
+            Optional. List of lists of strings to request capabilities. To use GPU, you need
```

Comment:
This parameter is confusing. Can we set a reasonable default so that most users will not have to fiddle with this?